styles/prosilver/template/pastebin_body.html

```diff
@@ -88,5 +66,27 @@
 </div>
 
 
+<!-- IF S_AUTH_VIEW -->
+<div class="mini-panel navbar">
+	<span class="corners-top"><span></span></span>
```

Comment:
mini-panel wird in der pastebin.css definiert. Ich bin mir nicht mal sicher ob das überhaupt jemals in phpBB drin war. Navbar gibts aber tatsächlich nicht.